fang_examples/simple_async_worker/src/lib.rs

```diff
@@ -69,3 +70,84 @@ impl AsyncRunnable for MyFailingTask {
         }
     }
 }
+
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "fang::serde")]
+pub struct MyCronTask {
+    pub number: u16,
+}
+
+impl MyCronTask {
+    pub fn new(number: u16) -> Self {
+        Self { number }
+    }
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "fang::serde")]
+pub struct MyTriggeredCronTasks {
+    pub number: u16,
+}
+
+impl MyTriggeredCronTasks {
+    pub fn new(number: u16) -> Self {
+        Self { number }
+    }
+}
+
+#[async_trait]
+#[typetag::serde]
+impl AsyncRunnable for MyTriggeredCronTasks {
+    async fn run(&self, queue: &mut dyn AsyncQueueable) -> Result<(), Error> {
+        let new_task = MyTriggeredCronTasks::new(self.number + 1);
+
+        queue
+            .insert_task(&new_task as &dyn AsyncRunnable)
+            .await
+            .unwrap();
+
+        log::info!(
+            "CRON!!!!!!!!!!!!!!!!!!!! the current number is {}",
+            self.number
+        );
+        tokio::time::sleep(Duration::from_secs(1)).await;
+
+        Ok(())
+    }
+
+    fn uniq(&self) -> bool {
+        true
+    }
+}
+
+#[async_trait]
+#[typetag::serde]
+impl AsyncRunnable for MyCronTask {
+    async fn run(&self, queue: &mut dyn AsyncQueueable) -> Result<(), Error> {
+        let new_task = MyTriggeredCronTasks::new(self.number + 1);
```

Comment:
remove task and  the insertion. cron task will be re-scheduled anyway

Reply:
[Comments adressed](https://github.com/ayrat555/fang/pull/71/commits/e32b0fe5b77adb1ff9ec8a9453335b37a00bfccc)